app/src/main/res/layout/activity_game.xml

```diff
@@ -68,13 +68,32 @@
             android:visibility="gone"
             android:layout_marginTop="24dp" />
 
-        <Button
-            android:id="@+id/songGuessSubmitButton"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
             android:layout_marginTop="24dp"
-            android:text="@string/submit"
-            android:textSize="16sp" />
+            android:orientation="horizontal">
+
+            <Button
+                android:id="@+id/songGuessSubmitButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/submit"
+                android:textSize="16sp"
+                style="?android:attr/buttonBarButtonStyle" />
+
+            <Button
+                android:id="@+id/showLyricsButton"
+                style="?android:attr/buttonBarButtonStyle"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:textSize="16sp"
+                android:text="@string/show_lyrics" />
```

Comment:
You should init its visibility to `gone` because otherwise, the first guesser sees this button even when he souldn't.

Reply:
Ok !